traffic_portal/app/src/common/modules/table/profilesParamsCompare/table.profilesParamsCompare.tpl.html

```diff
@@ -48,8 +48,8 @@
                 <td data-search="^{{::pp.name}}$">{{::pp.name}}</td>
                 <td data-search="^{{::pp.configFile}}$">{{::pp.configFile}}</td>
                 <td data-search="^{{::pp.value}}$">{{::pp.value}}</td>
-                <td class="center" data-order="{{pp.selected1}}"><input type="checkbox" ng-change="onChange(1)" ng-class="{'dirty': isCheckboxDirty(pp, 1)}" ng-model="pp.selected1"></td>
-                <td class="center" data-order="{{pp.selected2}}"><input type="checkbox" ng-change="onChange(2)" ng-class="{'dirty': isCheckboxDirty(pp, 2)}" ng-model="pp.selected2"></td>
+                <td class="center" data-order="{{pp.selected1}}"><input type="checkbox" ng-change="onChange(1)" ng-class="{'dirty': isCheckboxDirty(pp, 1)}" ng-model="pp.selected1"><span hidden>{{pp.selected1}}</span></td>
+                <td class="center" data-order="{{pp.selected2}}"><input type="checkbox" ng-change="onChange(2)" ng-class="{'dirty': isCheckboxDirty(pp, 2)}" ng-model="pp.selected2"><span hidden>{{pp.selected2}}</span></td>
```

Comment:
what's with the hidden span here?

Reply:
it's to hide the span :P

Reply:
it's because when the table is exported to csv, there is no value in this table cell. the export is not smart enough to figure out the value of the checkbox. instead, if i include this hidden field it is hidden from the UI but gets captured in the export.

Reply:
> _"the export is not smart enough to figure out the value of the checkbox"_

That sounds like a dirty hack to me, but I think you'd be disinclined to write it better yourself?

Reply:
it does feel hacky....but it is such a simple hack (so little code) i couldn't resist. plus, monkey patching the library felt more problematic than it's worth.

Reply:
Oh I didn't mean patching the library, I meant using the download stuff from the utils service to make your own button.